src/type/definition.d.ts

```diff
@@ -286,6 +286,7 @@ export type Thunk<T> = (() => T) | T;
 export class GraphQLScalarType {
   name: string;
   description: Maybe<string>;
+  specifiedByUrl?: Maybe<string>;
```

Comment:
```suggestion
  specifiedByUrl: Maybe<string>;
```
Since you always assign it inside constructor

Reply:
Fixed in 22524b102198e2b9c5103ebdf989eb6d069d2943